subiquity/models/network.py

```diff
@@ -15,6 +15,7 @@
 
 import logging
 
+from cloudinit import features
```

Comment:
So this is cloudinit from the snap, where we really want the features from the cloudinit on the target system which is a bit harder to access. It also makes things a bit more annoying because now (pedantically speaking) the result of NetworkModel.render now depends on the configuration of the source model (now I'm off wondering about how this would work when we support downloading the install artefacts over the network but well that's definitely one for the future)